### DISCUSSION TITLE
Visual studio code GIT problem

### DISCUSSION THREAD

**Discussion by vpanevv** [Cyfrin/foundry-full-course-cu#3109] [Q&A]
Hi all, I have a problem with Visual studio code push and CI procedure. I started a simple project with SimpleStorage from the course and when try do push the changes in GitHub, every time it gets to the same error that foundry looks for a test functions in the code and I can't make a pull request. Will be very thankful if someone can help me with this.

<img width="1405" alt="Screenshot 2024-12-27 at 13 41 56" src="https://github.com/user-attachments/assets/1bbefb5d-ebf7-4496-a9a0-f3fd5ac4bfee" />

**Comment by EngrPips** [comment]
I am unsure how you are running into the issue as I have never run it before, but why not create a simple test so that there will be some test to run and the process won't break at that point?

**Reply by vpanevv** [reply]
> I am unsure how you are running into the issue as I have never run it before, but why not create a simple test so that there will be some test to run and the process won't break at that point

At this time I just removed the run tests command in the github.yml file and that fix the problem. 
 ![image](https://github.com/user-attachments/assets/170837d0-ef72-4769-9e5a-ee01bc38cd97)

**Reply by EngrPips** [reply]
You are a Giga chad.
